pages/fixtures/combined.fixtures.ts

```diff
@@ -0,0 +1,34 @@
+import { test as base } from '@playwright/test';
```

Comment:
Fixtures could be extended. You can create your own structure. For example:
```suggestion
import { test as base } from 'loggedInPage.fixture.ts';
```
So you would have access to loggedInPage here, this will reduce of code duplication.